addons/mail/static/src/components/attachment/attachment.scss

```diff
@@ -22,7 +22,10 @@
 
 .o_Attachment_aside {
     position: relative;
-    overflow: hidden;
+
+    &.o-isInComposer {
+        overflow: hidden;
+    }
```

Comment:
What's the goal here? And why would we allow it to overflow in any other case?

Reply:
Actually I should reformulate the question, why is there an overflow hidden in the first place?

`overflow: hidden` should usually be avoided all-together.
The only legitimate use case off the top of by head is to be able to add ellipsis to text when we don't want it to wrap to new line.
The better alternative is to ensure elements that can get too big don't get too big in the first place, with a proper max-width, flex shrink, wrapping, or whatever else is necessary.

Reply:
@seb-odoo I think that the purpose was to hide the "big" cross button to delete attachments in the composer, it is hidden out of the card and it seems that it's a trick to make it slide in the card when we hover it.